OtripleS.Web.Api/Controllers/StudentSemesterCoursesController.cs

```diff
@@ -56,6 +56,45 @@ public class StudentSemesterCoursesController : RESTFulController
             }
         }
 
+        [HttpDelete("{studentId}/{semesterCourseId}")]
```

Comment:
This route should be `studentsemestercourses/students/{studentId}/semesters/{semesterId}`